packages/main/src/Panel.js

```diff
@@ -390,35 +376,29 @@ class Panel extends UI5Element {
 		return this.accessibleRole.toLowerCase();
 	}
 
+	get accessibleName() {
```

Comment:
Having a property in the metadata "accessibleName" and getter with the same name "accessibleName" might lead to issues. I suggest renaming the getter to _accessibleName or effectiveAccessibleName.

Reply:
done